flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -529,6 +529,30 @@ public void testDropIfExists() {
 		check(sql, "DROP TABLE IF EXISTS `CATALOG1`.`DB1`.`TBL1`");
 	}
 
+	@Test
+	public void testCreateFunction() {
+		String sql = "CREATE FUNCTION catalog1.db1.function1 AS 'org.apache.fink.function.function1'";
```

Comment:
need to add `CREATE FUNCTION func AS '...'` without `catalog`/`db`: e.g. test both fully qualified and just function  name?